includes/definitions/discovery/aos7.yaml

```diff
@@ -161,7 +167,8 @@ modules:
                         - { value: 1, generic: 1, graph: 1, descr: notRunning }
                         - { value: 2, generic: 0, graph: 1, descr: running }
                 -
-                    oid: chasEntPhysOperStatus
+                    oid: chasEntPhysicalEntry 
```

Comment:
Why are you walking the table now? doens't seem like you use anything else below.  Also please use chasEntPhysicalTable instead of chasEntPhysicalEntry

Reply:
This is even worse. I did it to see if this is changing the test results in a positive way. 
chasEntPhysicalTable  is bigger than chasEntPhysicalEntry. 

I will stick with `oid: chasEntPhysOperStatus`.

Reply:
chasEntPhysicalEntry chasEntPhysicalTable should have the same output because the table references the entry....